src/platform_impl/linux/event_loop.rs

```diff
@@ -312,6 +332,21 @@ impl<T: 'static> EventLoop<T> {
             window.set_skip_taskbar_hint(skip);
             window.set_skip_pager_hint(skip)
           }
+          WindowRequest::ProgressBarState(state) => {
```

Comment:
and move this logic down below next to global shortcut and menu handling

Reply:
?

Reply:
This? https://github.com/tauri-apps/tao/pull/742/commits/16db1576d701903801b4c07e5ca809ed918ef07c

Reply:
yeah